### PR TITLE
let zowe launch script to handle configure

### PR DESCRIPTION
**Pull request by jackjia-ibm**
Signed-off-by: Jack (T.) Jia <jack-tiefeng.jia@ibm.com>

This PR is part of https://github.com/zowe/zowe-install-packaging/issues/1769

## PR Type
- [ ] Bug fix
- [x] Feature
- [ ] Other (Please indicate)

## PR Checklist
- [ ] PR completes `npm run preCommit` without error
- [ ] Relevant Test cases have been added (Unit and or FVT)
- [ ] Relevant update to CHANGELOG.md
- [ ] PR from forked repo? Ensure Allow edits by maintaners is set.

### CODE REVIEW

**Comment by jackjia-ibm** [comment]
The integration test has been done https://github.com/zowe/zowe-install-packaging/pull/1849#issuecomment-746265697. This PR depends on https://github.com/zowe/zowe-install-packaging/pull/1849, must be merged after that. Please let me know if there are any questions. Thanks.

**Review by stevenhorsman — APPROVED** [review]

File: apiml-static-registration.yaml.template [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+- serviceId: explorer-uss
+  title: USS Explorer
+  description: IBM z/OS Unix Files UI service
+  catalogUiTileId:
+  instanceBaseUrls:
+  - https://${ZOWE_EXPLORER_HOST}:${USS_EXPLORER_UI_PORT}/
+  homePageRelativeUrl:
+  routedServices:
+  - gatewayUrl: ui/v1
+    serviceRelativeUrl: /ui/v1/explorer-uss
```
Comment (stevenhorsman) [inline apiml-static-registration.yaml.template:11]: I'm slightly concerned that we have this is 3 places now. If it's hardcoded here I'm not sure if it is needed in both manifest.yaml and package.json?

Comment (jackjia-ibm) [reply apiml-static-registration.yaml.template:11]: I found this is a little complicated to fix. We will also need a YAML parser to support parsing manifest in order to properly prepare FVT test environment. Right now we don't have a proper template, so are using the embedded environment variables as template format. I will come back to this later to solve this issue.

Comment (jackjia-ibm) [reply apiml-static-registration.yaml.template:11]: https://github.com/zowe/zlux/issues/567 created.

**Pull request merged**